faq.md

```diff
@@ -28,18 +28,18 @@ From _Tools_, select _Type Converter_ and enter the type descriptor in the box
 Follow the SOP: Investigating Failure to figure out the source of the problem. //TODO
 
 ## There was a thing that failed. How do I make it go away?
-If it's a Niassa workflow and you've relaunched the workflow, just wait. It will find it eventually. If impatient, you can purge it as below.
+If it's a Vidarr workflow, choose the command to retry the failed workflow.
 
-If you've changed the olive/LIMS/run blacklist/whatever to exclude this action, you can purge this action. From the Actions page, select a filter that finds your bad action, then click PURGE. If the world hasn't changed, the olive will recreate it.
+If you've changed the olive/LIMS/run blacklist/whatever to exclude this action, you can purge this action. From the _Actions_ page, select a filter that finds your bad action, then click _PURGE_. If the world hasn't changed, the olive will recreate it.
 
-Have at look at the Last Generated by an Olive age. If this is over 2 hours, you can be very confident the olive is no longer creating this action
+Have at look at the _Last Generated_ by an _Olive_ age. If this is over 2 hours, you can be very confident the olive is no longer creating this action
 
 ## I changed my olive but the actions are the same. What gives?
 Every time an olive runs, all the actions are generated and go into Shemu's scheduler. When the olive runs again, it regenerates all those actions and Shesmu needs to decide if two actions are the same. Actions get to decide if two actiosn are equivalent and that may not include all parameters. If you look at the actions, it will list the olives that generated and you will see two version of the same source file.
 
 To get Shesmu to replace the actions, purge the old ones (either individually or in bulk). This will cause the new action to be in the scheduler.
 
-For Niassa workflows, it uses the workflow SWID, the LIMS keys, and any annotations. If your olive only changes INI parameters, the modified actions will not displace the old actions. Even if purged, the new action will still match the existing Niassa workflow runs. If those need to be re-run, click the _Skip and Re-run_ button on the action.
+For Vidarr workflows, it uses the workflow, the input files, the LIMS keys, and labels. If your olive only changes arguments, the modified actions will not displace the old actions. Even if purged, the new action will still match the existing Vidarr workflow runs. If those need to be re-run, click the _Unload_ button on the action.
```

Comment:
There's `Delete and Purge` available for failed actions, but no `Unload` button available for SUCCEEDED actions. Perhaps remove this last sentence until we implement the `Unload` button?